subsys/bluetooth/host/hci_core.c

```diff
@@ -3387,6 +3387,20 @@ void hci_event_prio(struct net_buf *buf)
 	}
 }
 
+#if !defined(CONFIG_BT_RECV_IS_RX_THREAD)
+static void rx_queue_put(struct net_buf *buf)
+{
+	int err;
+
+	net_buf_put(&bt_dev.rx_queue, buf);
+
+	err = k_work_submit(&rx_work);
+	if (err) {
```

Comment:
```suggestion
	if (err < 0) {
```

k_work_submit will return 0, 1 or 2 on success :)